packages/presentation/lib/ui/movie_page/mapper/mapper_movie_list.dart

```diff
@@ -32,6 +32,7 @@ class MapperMovieList
       genres: movie.genres?.first.capitalize,
       runtime: movie.runtime!.formatter,
```

Comment:
never, just never use the bang operator. Only if u did some null check, such as in 31 line